packages/esm-patient-orders-app/src/config-schema.ts

```diff
@@ -6,15 +6,53 @@ export const configSchema = {
     _description: 'The encounter type of the encounter encapsulating orders',
     _default: '39da3525-afe4-45ff-8977-c53b7b359158',
   },
+  debounceDelayInMs: {
+    _type: Type.Number,
+    _description:
+      'Number of milliseconds to delay the search operation in the drug search input by after the user starts typing. The useDebounce hook delays the search by 300ms by default',
+    _default: 300,
+  },
   showPrintButton: {
     _type: Type.Boolean,
     _default: false,
     _description:
       'Determines whether or not to display a Print button in the Orders details table. If set to true, a Print button gets shown in both the orders table headers. When clicked, this button enables the user to print out the contents of the table',
   },
+  orderTypes: {
+    _type: Type.Array,
+    _default: [
+      // {
+      //   orderTypeUuid: '67a92e56-0f88-11ea-8d71-362b9e155667',
+      //   orderableConcepts: [],
+      // },
+      // {
+      //   orderTypeUuid: '67a9328e-0f88-11ea-8d71-362b9e155667',
+      //   orderableConcepts: [],
+      // },
+    ],
+    _elements: {
+      orderTypeUuid: {
+        _type: Type.String,
+        _description: 'Order type UUID to be displayed on the order basket',
+      },
+      orderableConcepts: {
```

Comment:
For historical reasons, I don't love the idea of getting all concepts with a given concept class as orderables. Our metadata management hasn't been disciplined enough to make that useful.

Sidebar, does this also support `orderableConceptSets` where the members themselves are concept sets? This is helpful for things like lab test results, and I think that's a pattern we'll want with, e.g., procedure orders at the very least.

Reply:
Hi Ian!

> I don't love the idea of getting all concepts with a given concept class as orderables.

If the implementer mentions the concept sets, then it'll be given higher priority than the conceptClass if mentioned.

> does this also support orderableConceptSets where the members themselves are concept sets

Currently not, but I can push the change for that.

Reply:
To be clear about this, `conceptClass` for this is, at best, a kludge—and really an abuse of concept metadata. This is because there are always multiple ways concepts can be classified. For example, from the perspective of an EMR, a radiology test is... just a test the same way as a lab test is. It has a different workflow, etc., but it's still just a test. 

The bigger issue here, though, is that relying on concept class to distinguish types of orderables requires metadata that is not present in CIEL and hence will not be present in most implementations—and over the long term, relying on things that are not part of the default community-supported thing is a great way to end up relying on a feature that will not be long-term sustainable.

Reply:
I was going to say that I didn't mind conceptClass as a fallback method as long as conceptSets is the preferred method but @ibacher  seems to be making a strong case for not supporting concept classes, adding @mseaton  for his thoughts.

As a tangential rabbit hole, I was looking at the O2 Lab configuration, and it currently is configured via single concept set *but* this is a set of sets, where each member is the a lab order category, and within each category are the orderables... and then within the orderables, some of these are setsrepresent a panel:

![image](https://github.com/user-attachments/assets/3b1d7c34-e3f5-4f42-aad5-1b329ad7f065)

For instance, in the above, the categories (Hematology, Biochemstry, etc) are the concepts that are members of the "Lab Orderables" concept set.

Then all the tests listed in "Panels" and "Tests" are the concepts that are members of the "Hematology Concept Set".  

And within that "Tests" are stand-alone tests, while "Panels" are concept sets that contain all the tests within that panel.

Not saying we need to do this now (especially since the O3 Lab Orders functionality is already in production ) but just an FYI while we are working on this that this is something we will need to support in the future for O2 compatibility (basically: 1) breaking the labs up into categories and 2) allow ordering tests as a panel or individually) so if we are expanding this new functionality to support "sets of concept sets" we should take this into consideration when designing.

Sorry for the long post!

Reply:
Hi @mogoodrich!
The above implementation can be used in the O3 as well, you can list both the `Hematology Concept Set` and `Lab Orderables` in the orderableConceptSets, it's just that it'll come in a single list, rather than the above grouping.

@ibacher, should I go ahead and remove the implementation supporting the conceptClasses?